### DISCUSSION TITLE
Is i18n supported?

### DISCUSSION THREAD

**Discussion by ichenhe** [thegeeklab/hugo-geekdoc#182] [Q&A] [answered]
It seems that there is no option for multilingual switching. I'm not sure if it's related to the theme, or do I have to build them by myself?

**Comment by xoxys** [comment, marked as answer]
Right now we don't have multi-language support added.

**Reply by ichenhe** [reply]
Thanks for your reply. It would be great if we could support it in the future.

**Reply by xoxys** [reply]
It's definitely planned to add :), and I would be happy to review a PR, personally I don't have the time to implement it in the next weeks.
